aten/src/ATen/native/TensorShape.cpp

```diff
@@ -6,7 +6,6 @@
 #include <ATen/NamedTensorUtils.h>
 #include <ATen/core/DimVector.h>
 #include <ATen/native/Copy.h>
-#include <ATen/native/cpu/CatKernel.h>
```

Comment:
See line 12 below. This was included twice.

Reply:
Nice attention to detail